README.md

```diff
@@ -187,6 +187,37 @@ for grabber in grabbers.values():
     grabber.release()
 ```
 
+#### RTSP Discovery
+RTSP cameras with support for ONVIF can be discovered using the tool built into the library. It provides a convenient method for finding all the RTSP cameras in the local network.
+
+```python
+from framegrab import RTSPDiscovery, ONVIFDeviceInfo
+        
+devices = RTSPDiscovery.discover_camera_ips()
+```
+
+The `discover_camera_ips()` will provide a list of devices that it finds in the `ONVIFDeviceInfo` format. An optional flag `try_default_logins` can be used to try different default credentials to fetch RTSP URLs.
+
+```python
+# Model for storing ONVIF Device Information
+class ONVIFDeviceInfo(BaseModel):
```

Comment:
Not sure if we need to show the end user the pydantic model, but perhaps it could be useful. If we show it, we should include `from pydantic import BaseModel` (or whatever it is) so that they know what we are showing them.